bindings/matrix-sdk-crypto-js/src/identities.rs

```diff
@@ -85,6 +85,28 @@ impl OwnUserIdentity {
 
         future_to_promise(async move { Ok(me.trusts_our_own_device().await?) })
     }
+
+    /// Get the master key of the identity.
+    #[wasm_bindgen(getter, js_name = "masterKey")]
+    pub fn master_key(&self) -> Option<String> {
+        let master_key = self.inner.master_key().as_ref().to_owned();
+        serde_json::to_string(&master_key).ok()
+    }
+
+    /// Get the self-signing key of the identity.
+    #[wasm_bindgen(getter, js_name = "selfSigningKey")]
+    pub fn self_signing_key(&self) -> Option<String> {
+        let self_signing_key = self.inner.master_key().as_ref().to_owned();
```

Comment:
You might want to add tests to check that the `self_signing_key` method doesn't return the wrong key since we're converting to a stringly typed API here.

```suggestion
           let self_signing_key = self.inner.self_signing_key().as_ref();
```

Reply:
Yup, the JS bindings are fairly well tested: https://github.com/matrix-org/matrix-rust-sdk/tree/main/bindings/matrix-sdk-crypto-js/tests.

Reply:
Added some tests to check if the keys are defined and they are different